operations/host_spawn.go

```diff
@@ -1375,6 +1375,61 @@ Examples:
 	}
 }
 
+func hostFindWithIpAddress() cli.Command {
+	const (
+		ipAddressFlagName = "ip-address"
+		spruceFlagName    = "spruce"
+	)
+	return cli.Command{
+		Name:  "find-by-ip-address",
```

Comment:
Users would need to type out `evergreen find-by-ip-address --ip-address <address>` which is redundant. Can we just call this get-host or something? This would also allow us to add other forms of lookup in the future (may also want to change the usage description).